modules/ipi-install-configuring-storage-on-nodes.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-storage-on-nodes_{context}"]
+= Optional: Configuring storage on nodes
+
+You can make changes to operating systems on {product-title} nodes by creating `MachineConfig` objects that are managed by the Machine Config Operator (MCO).
+
+The `MachineConfig` specification includes an ignition config for configuring the machines at first boot. This config object can be used to modify files, systemd services, and other operating system features running on on {product-title} machines.
+
+.Procedure
+
+Use the ignition config to configure storage on the nodes. The following `MachineSet` manifest example demonstrates how to add a partition to the sda device on primary nodes. In this example, apply the manifest before installation to have a partition named `recovery` with a size of 16 GiB on the primary nodes.
+
+. Create the new `MachineConfig` manifest as shown in the following example:
++
+[source,terminal]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: primary
+  name: 10_primary_storage_config
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      disks:
+        - device: </dev/xxyN>
+          partitions:
+            - label: recovery
+              startMiB: 32768
+              sizeMiB: 16384
+      filesystems:
+        - device: /dev/disk/by-partlabel/recovery
+          label: recovery
+          format: xfs
+----
++
+. Save and copy the new `MachineConfig` manifest to the `clusterconfigs/openshift` directory by running the following command:
++
+[source,terminal]
+----
+$ cp ~/<MachineConfig_manifest> clusterconfigs/openshift
```

Comment:
this assumes that manifest is in your home dir.
maybe better 
cp  <machineConfigmanifest> ~/clusterconfigs/openshift ?

Reply:
Updated As Suggested.